Vagrantfile

```diff
@@ -23,6 +23,16 @@ BAZEL_VERSION = ENV['BAZEL_VERSION']
 $bootstrap = <<SCRIPT
 echo "----------------------------------------------------------------"
 export PATH=/home/vagrant/go/bin:/usr/local/clang/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games
+
+echo "editing journald configuration"
+sudo bash -c "echo "RateLimitIntervalSec=1s" >> /etc/systemd/journald.conf"
```

Comment:
The quoting is funky.
It's harmless, but I'd remove the double quotes around `RateLimitIntervalSec=1s`

Reply:
Fixed in latest upload.